project_report/project_information.py

```diff
@@ -0,0 +1,490 @@
+    def get_from_project_summary_yaml(summary_yaml):
+        with open(summary_yaml, 'r') as open_file:
+            full_yaml = yaml.safe_load(open_file)
+        sample_yaml = full_yaml['samples'][0]
+        return path.basename(path.dirname(sample_yaml['dirs']['galaxy'])), sample_yaml['genome_build']
+
+    def get_bcl2fastq_version(self, run_ids):
+        bcl2fastq_versions = set()
+        for run_id in run_ids:
+            prog_vers_yaml = os.path.join(self.run_folders, run_id, 'program_versions.yaml')
+            with open(prog_vers_yaml, 'r') as open_file:
+                full_yaml = yaml.safe_load(open_file)
+                if 'bcl2fastq' in full_yaml and full_yaml.get('bcl2fastq'):
+                    bcl2fastq_versions.add(full_yaml.get('bcl2fastq'))
+                else:
+                    self.warning('Run %s has no bcl2fastq version: default to v2.17.1.14', run_id)
+                    bcl2fastq_versions.add('v2.17.1.14')
+        return ', '.join(bcl2fastq_versions)
+
+    def update_from_program_version_yaml(self, prog_vers_yaml):
+        with open(prog_vers_yaml, 'r') as open_file:
+            full_yaml = yaml.safe_load(open_file)
+            for p in ['bwa', 'gatk', 'samtools', 'samblaster', 'biobambam_sortmapdup']:
+                if p in full_yaml:
+                    self.params[p + '_version'] = full_yaml.get(p)
+
+    def get_project_info(self):
+        self.store_sample_info()
+        species_submitted = set()
+        library_workflows = self.get_library_workflows()
+        for sample in self.sample_names_delivered:
+            species = self.get_species_from_sample(sample)
+            species_submitted.add(species)
+        project_info = (
+            ('Project name', self.project_name),
+            ('Project title', self.project_title),
+            ('Enquiry no', self.enquiry_number),
+            ('Quote no', self.quote_number),
+            ('Customer name', self.customer_name),
+            ('Customer address', '\n'.join(self.customer_address_lines)),
+            ('Number of samples', self.number_quoted_samples),
+            ('Number of samples delivered', len(self.samples_for_project_restapi)),
+            ('Date samples received', 'Detailed in appendix I'),
+            ('Total download size', '%.2f terabytes' % self.project_size_in_terabytes()),
+            ('Laboratory protocol', ', '.join(library_workflows)),
+            ('Submitted species', ', '.join(list(species_submitted))),
+            ('Genome version', self.params['genome_version'])
+        )
+
+        return project_info
+
+    @staticmethod
+    def min_mean_max(list_values):
+        if list_values:
+            return 'min: %.1f, avg: %.1f, max: %.1f' % (
+                min(list_values),
+                ProjectReportInformation.calculate_mean(list_values),
+                max(list_values)
+            )
+        else:
+            return 'min: 0, mean: 0, max: 0'
+
+    def calculate_project_statistsics(self):
```

Comment:
```suggestion
    def calculate_project_statistics(self):
```

Reply:
Has this been corrected please? It still seems to be "calculate_project_statistsics" on my end.